_playbooks/rpa.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: playbook
+title: Robotics Process Automation
+description: This is the RPA playbook 
```

Comment:
@Bkim0928 could you update this description to something a bit more complete? It shows up on the [playbook listing](https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/gsa/cto-website/rpa-bk/playbooks/).

Reply:
Updated description! Also added the RPA '.png' file to assets (it was missing)